src/qt/forms/sendcoinsdialog.ui

```diff
@@ -201,6 +201,9 @@
              </property>
              <item>
               <layout class="QFormLayout" name="formLayoutCoinControl1">
+               <property name="labelAlignment">
+                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
```

Comment:
I'm not too experienced with Qt, can anyone elaborate on that question (ping @furszy as author of the second commit)? It seems at least that we already have a couple of instances in master where both Qt::AlignLeading and Qt::AlignLeft are used together:
```
$ git grep AlignLeading
src/qt/forms/debugwindow.ui:              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
src/qt/forms/helpmessagedialog.ui:        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
src/qt/forms/helpmessagedialog.ui:            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
src/qt/forms/signverifymessagedialog.ui:          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>

```

Reply:
> It seems at least that we already have a couple of instances in master where both Qt::AlignLeading and Qt::AlignLeft are used together.

Quite sure that we have them because of QT creator setting the value automatically. Probably because there is some ancient qt version that doesn't have them as synonyms.

But for us, it should be fine to drop them, even qt 4.8 has them as synonyms ([link](https://het.as.utexas.edu/HET/Software/html/qt.html#AlignmentFlag-enum)).

Reply:
@furszy: Thanks a lot for checking. I removed the redundant `Qt::AlignLeading` properties from the second commit, i.e. it's only `Qt::AlignLeft` remaining.